pkg/controller/csi_handler.go

```diff
@@ -724,7 +725,7 @@ func (h *csiHandler) patchVA(va, clone *storage.VolumeAttachment) (*storage.Volu
 		return va, err
 	}
 
-	newVa, err := h.client.StorageV1beta1().VolumeAttachments().Patch(va.Name, types.MergePatchType, patch)
+	newVa, err := h.client.StorageV1().VolumeAttachments().Patch(va.Name, types.MergePatchType, patch)
```

Comment:
if this is changing status fields, it must submit the patch to the status subresource